src/entities/comment/libs/hooks/useCommentsGet.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback, useContext } from "react";
```

Comment:
- move this file to `src/entities/comment/api/hooks`
- rename file to `useFetchCommentsByTwinId`
- rename function `getComments` to `fetchComments`
- rename `id` argument into `twinId` 
-